app/views/auth_sources/_welcome.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="blank-slate-pf">
```

Comment:
I think with previous version of this PR, we were not using this file at all and it was dead code. @sharvit thanks for the `nit` related to partial, it helped me to identify this and use it.

Let me know if what I have done is [correct.](https://github.com/theforeman/foreman/pull/7206/files#diff-4e0b11077934d88147beb7787580bd32R20)

Reply:
Nice :+1: